src/renderers/shared/fiber/ReactFiberCommitWork.js

```diff
@@ -193,11 +195,14 @@ module.exports = function<T, P, I, TI, C>(
         commitNestedUnmounts(node);
```

Comment:
Does this call (or any `commitUnmount()` calls it makes) also need to be guarded by `!ignoreTop`?
I don't fully understand what's happening here but it offers another path to `commitUnmount()`.